_tools/lint_plugin.ts

```diff
@@ -310,5 +312,58 @@ export default {
         };
       },
     },
+    // https://docs.deno.com/runtime/contributing/style_guide/#exported-functions%3A-max-2-args%2C-put-the-rest-into-an-options-object
+    "exported-function-args-maximum": {
+      create(context) {
+        const url = toFileUrl(resolve(context.filename));
```

Comment:
Might `import.meta.resolve()` work here?

Reply:
`context.filename` is `main.ts` in test cases, and that throws with `import.meta.resolve` (`main.ts` is not a valid specifier, but a valid file path)